libs/dao-data/src/HausError.ts

```diff
@@ -1,8 +1,12 @@
-export type ErrorType = 'SUBGRAPH_ERROR' | 'UNSUPPORTED_NETWORK';
+export type ErrorType =
+  | 'SUBGRAPH_ERROR'
+  | 'UNSUPPORTED_NETWORK'
+  | 'REQUEST_ERROR';
 
 const errorTypeToTitleMap = new Map<ErrorType, string>([
```

Comment:
Nice! That's a super cool pattern!

Reply:
> Nice! That's a super cool pattern!

It's a little bit of a shot in the dark. I was thinking we ought to add a ticket to look at the error handling patterns and define that for the whole library.